curs_main.c

```diff
@@ -50,6 +50,7 @@
 #include "protos.h"
 #include "sort.h"
 #include "thread.h"
+#include <term.h>
```

Comment:
It exists in both curses and slang, but in different headers.
In curses, `tigetstr()` is in `<term.h>`, but in slang it's (a null op) in `slcurses.h`